.env.template

```diff
@@ -61,4 +61,12 @@ AWS_GENO_BUCKET=<s3 bucket for genotypic data uploads>
 AWS_S3_ENDPOINT=<s3 endpoint, default https://s3.us-east-1.amazonaws.com>
 
 BRAPI_VENDOR_SUBMISSION_ENABLED=false #can a submission be sent to a vendor via BrAPI
-BRAPI_VENDOR_CHECK_FREQUENCY=1d #how often to check for vendor updates for sample submissions
\ No newline at end of file
+BRAPI_VENDOR_CHECK_FREQUENCY=1d #how often to check for vendor updates for sample submissions
+
+#The initial caching of each type of object needs to be staggered by the prescribed number of seconds
```

Comment:
Could you make these new envs all caps to be consistent with the others?

Reply:
updated to ALL CAPS